CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 - master
   - New
   - Changed
+    - Fix a badchar in progress output ([#390](https://github.com/ffuf/ffuf/pull/390), thx to [@noraj](https://github.com/noraj))
```

Comment:
```suggestion
    - Fix a badchar in progress output
```

Reply:
I would prefer keeping `CHANGELOG.md` just as a changelog. The shoutouts are done in `CONTRIBUTORS.md` ;)